saleor/settings.py

```diff
@@ -231,6 +240,20 @@ def get_url_from_env(name, *, schemes=None) -> Optional[str]:
     "saleor.core.middleware.jwt_refresh_token_middleware",
 ]
 
+# Database alias logger middleware
+ENABLE_DB_ALIAS_LOGGER_MIDDLEWARE = get_bool_from_env(
+    "ENABLE_DB_ALIAS_LOGGER_MIDDLEWARE", False
+)
+if ENABLE_DB_ALIAS_LOGGER_MIDDLEWARE:
+    MIDDLEWARE.append("saleor.core.db.connection.db_alias_logger_middleware")
```

Comment:
```suggestion
    MIDDLEWARE = ["saleor.core.db.connection.db_alias_logger_middleware"] + MIDDLEWARE
```
Shouldn't we add this middleware as the first one to also wrap other middleware potentially?